packages/devextreme-scss/scss/widgets/base/chat/layout/chat-messagebox/_mixins.scss

```diff
@@ -0,0 +1,28 @@
+@mixin chat-messagebox(
+  $padding,
+  $border-color,
+  $gap,
+  $textarea-min-height,
+  $textarea-border-radius,
+  $button-size
+) {
+  .dx-chat-message-box {
+    padding: $padding;
+    border-top: 1px solid $border-color;
+    gap: $gap;
+  }
+
+  .dx-chat-message-box-text-area {
+    &.dx-textarea.dx-texteditor {
+      min-height: $textarea-min-height;
+      border-radius: $textarea-border-radius;
+    }
+  }
+  
+  .dx-chat-message-box-button {
```

Comment:
Is it correct?

<img width="42" alt="image" src="https://github.com/user-attachments/assets/ffa93214-3b7d-4e35-8f10-a57b3a8a6c1b">

Reply:
yes